js/base/Exchange.js

```diff
@@ -1672,8 +1672,8 @@ module.exports = class Exchange {
         return decimalToPrecision (fee, ROUND, market.precision.price, this.precisionMode, this.paddingMode)
     }
 
-    currencyToPrecision (currency, fee) {
-        return decimalToPrecision (fee, ROUND, this.currencies[currency]['precision'], this.precisionMode, this.paddingMode);
+    currencyToPrecision (currencyCode, fee) {
```

Comment:
I would suggest currency > code to keep it in line with the other methods operating with currencies and currency codes.